pages/components/SignedUpComplete/SignedUpComplete.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+
+const SignedUpComponent = () => {
```

Comment:
It's good practice to name this the same name as your file; in this case, change it to "RegistrationComplete" and the same for the file name to better match the rest of the website.